src/cratedb_about/hub/model.py

```
@@ -0,0 +1,68 @@
+        divs = card.find(name="div")
+        title = divs.find(name="h3").text
+        tags = []
+        for tag in divs.find_all(name="span"):
+            tags.append(tag.text)
+        website = divs.find(name="p", attrs={"class": "text-sm"}).text
+        description = divs.find(name="p", attrs={"class": "text-sm", "title": True}).text
+        logo_url = divs.find(name="img").get("src")
+        resources = []
+        for anchor in divs.find_all(name="a"):
+            resources.append(Resource(url=anchor.get("href")))
+        return LLMsTxtHubItem(title=title, website=website, description=description, logo=logo_url, tags=tags, resources=resources)
```

Ruff (0.11.9)

Line too long (131 > 100)

(E501)